### PR TITLE
feat(components): add sidebar separator component

### PR DESCRIPTION
**Pull request by robinmetral**
## Purpose

Add a horizontal separator component to the Sidebar. This will be used to group items together and is particularly useful when information architectures become more complex.

Screenshot_2019-12-05 Components Sidebar - Base ⋅ Storybook(1)

## Approach and changes

Add a styled component in the Sidebar that extends the Circuit `Hr` component, just changing the color.

## Definition of done

* [x] Development completed
* [x] Reviewers assigned
* [x] Unit and integration tests
* [x] Meets minimum browser support
* [x] Meets accessibility requirements

### CODE REVIEW

File: src/components/Sidebar/components/Separator/Separator.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * limitations under the License.
+ */
+
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import Hr from '../../../Hr';
+
+const baseStyles = ({ theme }) => css`
+  label: separator;
```
Comment (connor-baer) [inline src/components/Sidebar/components/Separator/Separator.js]: This should be prefixed with the main component name: `sidebar-separator`.

Comment (robinmetral) [reply src/components/Sidebar/components/Separator/Separator.js]: Thanks!

Comment (robinmetral) [reply src/components/Sidebar/components/Separator/Separator.js]: Done in 3b9d93f

**Pull request merged**